src/convenience/frameworks.ts

```diff
@@ -160,7 +160,7 @@ const http: FrameworkAdapter = (req, res) => {
             const chunks: Chunk[] = [];
             req.on("data", (chunk: Chunk) => chunks.push(chunk))
                 .once("end", () => {
-                    // @ts-ignore `Buffer` is Node-only
+                    // @ts-expect-error `Buffer` is Node-only
```

Comment:
backport fails with this line as its no error in this case. Maybe revert this one back to ts-ignore and keep the other ones at ts-expect-error?

Reply:
Yes please